packages/forms-web-app/src/views/register/agent/cya/cya-representee-detail.njk

```diff
@@ -39,8 +35,9 @@
                 }
               ]
             }
-          },
-          {
+          }] %}
+{% if data.representing !== 'organisation' %}
+  {% set rows = (rows.push({
```

Comment:
rows.push can take an array of objects so no need to keep calling rows.push for each individual object after the conditional one (over 18)